core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -51,6 +52,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-disown"
     )
 
+    conf.sslConf match {
+      case Some(ssl) => {
+        cmdToLaunch ++ Array("-internal_security_config", ssl)
```

Comment:
I would keep it simple and wouldn't create another extra endpoint for this. As you pointed out, it would be unsecured anyway. We can just mention that this is expected behaviour for this deployment strategy and that the solution with h2o on yarn can handle in in more native way